internal/healthchecks/ports_check.go

```diff
@@ -45,6 +45,18 @@ func checkIfPortAvailable(host string, port string, network string) (bool, error
 }
 
 func (c PortsCheck) RunCheck(logger *log.Logger) error {
+	fbActive, err := isSubagentActive("google-cloud-ops-agent-fluent-bit")
```

Comment:
I don't think we should merge them, because there exist intermediate states (especially on Windows) where only one subagent is running. Each subagent uses a different port so the checks divide pretty cleanly anyway.